plugins/marvin.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: marvin
+spec:
+  homepage: https://github.com/undistro/marvin
+  shortDescription: Scans your cluster for potential issues. Supports custom checks with CEL.
+  description: |
+    Marvin scans a k8s cluster by performing CEL expressions 
```

Comment:
You can spell out Kubernetes, no need to abbreviate.

Reply:
Done